src/ShaderCache.cpp

```diff
@@ -1416,7 +1416,7 @@ namespace SIE
 					descriptor);
 			} else {
 				std::array<size_t, 3> bufferSizes = { 0, 0, 0 };
-				std::fill(newShader->constantTable.begin(), newShader->constantTable.end(), 0);
+				std::fill(newShader->constantTable.begin(), newShader->constantTable.end(), static_cast<uint8_t>(0));
```

Comment:
`std::ranges::fill(newShader->constantTable, static_cast<uint8_t>(0));` confirmed complies for me